pkg/apiserver/certificate/certificate.go

```diff
@@ -71,21 +55,22 @@ func ApplyServerCert(selfSignedCert bool,
 	client kubernetes.Interface,
 	aggregatorClient clientset.Interface,
 	apiExtensionClient apiextensionclientset.Interface,
-	secureServing *options.SecureServingOptionsWithLoopback) (*CACertController, error) {
+	secureServing *options.SecureServingOptionsWithLoopback,
+	caConfig CAConfig) (*CACertController, error) {
```

Comment:
`caConfig` should be a pointer conventionally unless you are making a copy intentionally.

Reply:
any comment for this one? I can see the type is passed to many functions, meaning the struct will be copied many times.

Reply:
ah, sure, I missed this comment, will make it as pointer.

Reply:
done